config/migrations/20220419144231-migratie-kaleidos-naar-themis-vergaderactiviteit-types.sparql

```diff
@@ -0,0 +1,98 @@
+PREFIX kkind: <http://kanselarij.vo.data.gift/id/concept/ministerraad-type-codes/>
+PREFIX tkind: <http://themis.vlaanderen.be/id/concept/vergaderactiviteit-type/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX schema: <http://schema.org/>
+
+DELETE WHERE {
+  GRAPH ?g {
+    <http://kanselarij.vo.data.gift/id/concept-scheme/668f0c19-3864-4e96-9dd3-de806fd53357> ?p00 ?o00 .
+    kkind:A5D6B7A8-2F9C-44B6-B3BE-98D80B426254 ?p11 ?o11 . # Ministerraad
+    kkind:406F2ECA-524D-47DC-B889-651893135456 ?p12 ?o12 . # Elektronische procedure
+    kkind:7D8E35BE-E5D1-494F-B5F9-51B07875B96F ?p13 ?o13 . # Bijzondere ministerraad
+    kkind:1d16cb70-0ae9-489e-bf97-c74897222e3c ?p14 ?o14 . # Ministerraad - Plan Vlaamse Veerkracht
+    kkind:d36138a9-07f0-4df6-bbf0-abd51a24e4ce ?p15 ?o15 . # Annex
+  }
+};
+
+DELETE {
```

Comment:
Can you try to rewrite these `DELETE...INSERT...WHERE` queries to one query using a `VALUES ?kaleidosKind ?themisKind { .... }` block? Makes it easier to understand what's happening.